src/pages/shared/ModlogPage.tsx

```diff
@@ -44,13 +58,34 @@ const ModlogByCommunityName = memo(function ModlogByCommunityName({
   communityName: string;
 }) {
   const community = useFetchCommunity(communityName);
+  const canModerate = useCanModerate(community?.community);
+  const user = useAppSelector(userSelector);
 
   if (!community) return <CenteredSpinner />;
 
-  return <ModlogByCommunity community={community.community} />;
+  if (canModerate) return <Modlog community={community.community} />;
+  return <Modlog community={community.community} user={user} />;
 });
 
-function ModlogByCommunity({ community }: { community?: Community }) {
+const ModlogByUserHandle = memo(function ModlogByUserHandle({
+  handle,
+}: {
+  handle: string;
+}) {
+  const user = useUserDetails(handle);
```

Comment:
You could probably just make this a selector, there's a lot of unecessary stuff in useUserDetails.

Reply:
It looks like the only thing I need from that hook is this:

```ts
const userByHandle = useAppSelector((state) => state.user.userByHandle);
const user = userByHandle[handle];
```

Can I combine the two into a single line like this:

```ts
const user = useAppSelector((state) => state.user.userByHandle[handle]);
```

...or is that liable to break Redux somehow? I just tested it and it DOES seem to work.